sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -482,9 +482,17 @@ def test_combining_with_accumulation_mode_and_fanout(self):
               trigger=AfterWatermark(early=AfterAll(AfterCount(1))))
           | beam.CombineGlobally(sum).without_defaults().with_fanout(2))
 
-      # The frings for DISCARDING mode is [1, 2, 3, 4, 5, 0, 0].
-      firings = [1, 3, 6, 10, 15, 15, 15]
-      assert_that(result, equal_to(firings))
+      def has_expected_values(actual):
+        from hamcrest.core import assert_that as hamcrest_assert
+        from hamcrest.library.collection import only_contains
+        ordered = sorted(actual)
+        # Early firings.
+        hamcrest_assert(ordered[:4], only_contains(1, 3, 6, 10))
```

Comment:
`only_contains` allows for duplicates, which you don't want here. Use `contains` instead. (You could use it below, and drop the `set` in `set(ordered[4:])`.)

Reply:
Done. Thank you! I forgot that `contains` exactly match the entire sequence